codelists/tests/test_actions.py

```diff
@@ -425,6 +428,164 @@ def test_export_to_builder(organisation_user, new_style_version):
+    ..  202855006       │ │   └ Lateral epicondylitis
+    +   35185008        │ ├ Enthesopathy of elbow region
+    (+) 73583000        │ │ └ Epicondylitis
+    (+) 202855006       │ │   └ Lateral epicondylitis
+    ..  239964003       │ └ Soft tissue lesion of elbow region
+    ..  298869002       └ Finding of elbow joint
+    ..  429554009         ├ Arthropathy of elbow
+    ..  439656005         │ └ Arthritis of elbow
+    ..  202855006         │   └ Lateral epicondylitis
+    ..  298163003         └ Elbow joint infla
+    ..  439656005           └ Arthritis of elbow
+    ..  202855006             └ Lateral epicondylitis
+    ..  238484001       Tennis toe
+    +   156659008       (Epicondylitis &/or tennis elbow) or (golfers' elbow) [inactive]
+    """
+
+    # Searches for "tennis toe" and "Enthesopathy of elbow" will return these 4 codes
+    initial_code_to_status = {
+        "238484001": "+",
+        "73583000": "(+)",
+        "202855006": "(+)",
+        "35185008": "+",
+    }
+    assert minimal_version.codeset.code_to_status == initial_code_to_status
+
+    # Set up the test scenarios
+    # Manipulate the code objs to represent a previous state of the coding system
+
+    # 1) 202855006 is a new code that will be returned when the searches are re-run
+    # i.e. in previous version
+    # +   73583000        │ ├ Epicondylitis
+    # (+) 35185008        │ │ └ Enthesopathy of elbow region
+    # in new version
+    # +   35185008        │ ├ Enthesopathy of elbow region
+    # (+) 73583000        │ │ └ Epicondylitis
+    # (+) 202855006       │ │   └ Lateral epicondylitis
+
+    # SET UP: Delete 202855006 from the version
+    CodeObj.objects.get(version=minimal_version, code="202855006").delete()
+    # This is the state of the codeset's code_to_status based on searches
+    # that did NOT return 202855006
+    assert minimal_version.codeset.code_to_status == {
+        "238484001": "+",
+        "73583000": "(+)",
+        "35185008": "+",
+    }
+
+    # update the draft to recreate its searches and derived statuses
+    cache.clear()
+    _, updates = actions.update_draft(draft=minimal_version)
+    # the "new" code has been set to its expected inherited status
+    assert minimal_version.codeset.code_to_status == initial_code_to_status
+    assert updates == {
+        "added": {("202855006", "(+)")},
+        "changed": set(),
+        "removed": set(),
+    }
+
+    # 2) Assume that 73583000 (Epicondylitis) and 35185008 (Enthesopathy of elbow region)
+    # have swapped places - ie. 73583000 used to be the parent of 35185008 and 202855006,
+    # and was explicitly included, with 35185008 implicitly included
+
+    # i.e. in previous version
+    # +   73583000        │ ├ Epicondylitis
+    # (+) 35185008        │ │ └ Enthesopathy of elbow region
+    # (+) 202855006       │ │   └ Lateral epicondylitis
+    # in new version
+    # +   35185008        │ ├ Enthesopathy of elbow region
```

Comment:
This comment doesn't match the assertion on line 539.

Reply:
"new version" here is the changes made in the test setup, not what it looks like after the updates.  That's not clear, I'll reword